### PR TITLE
Update Toon docs

### PR DESCRIPTION
**Pull request by Boltgolt**
**Description:**

Added more information to the Toon docs, and updated the titles so the components don't all show up as "Toon" in the component search.

### CODE REVIEW

**Comment by Boltgolt** [comment]
I see @fabaff also noticed the title issue and fixed it [here](https://github.com/home-assistant/home-assistant.github.io/pull/3752/files), but i think these titles are a lot more discriptive of what the component actially does.

File: source/_components/climate.toon.markdown [hunk 1]
```diff
@@ -13,6 +13,15 @@ logo: toon.png
 ha_iot_class: "Cloud Polling"
 ---
 
-The `toon` climate platform allows you to interact with your Toon device.
+The `toon` climate platform allows you to interact with your Toon thermostat. For compatibility reasons the states are different from the normal "Toon programma's", they are mapped as follows:
```
Comment (frenck) [inline source/_components/climate.toon.markdown]: I like the addition but maybe we should keep the documentation as English as possible.
I like to suggest: `For compatibility reasons, the states in Home Assistant are different from the states displayed on your Toon device and are mapped as follows:`

File: source/_components/climate.toon.markdown [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Toon"
-description: "Instructions how to integrate Toon Climate devices within Home Assistant."
-date: 2017-10-21 12:00
+title: "Toon Thermostat"
+description: "Instructions how to integrate Toon Thermostats within Home Assistant."
```
Comment (frenck) [inline source/_components/climate.toon.markdown]: `Thermostats` is not part of the title or product name and should be written lower caps.

File: source/_components/switch.toon.markdown [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Toon"
-description: "Instructions how to integrate Toon Climate devices within Home Assistant."
-date: 2017-10-21 12:00
+title: "Toon Smart Plugs"
+description: "Instructions how to integrate Toon Smart Plugs within Home Assistant."
```
Comment (frenck) [inline source/_components/switch.toon.markdown]: `Smart Plugs` is not part of the product name and should be written in lower caps.

**Comment by frenck** [comment]
@Boltgolt, Good job on the additions, nevertheless I have some small advise and concerns. Could you take a look?

**Comment by Boltgolt** [comment]
@frenck Thanks for the quick review, i pushed the changes as suggested.

**Comment by fabaff** [comment]
Those changes should have been made against `current`. #3752 introduced some changes.

**Pull request merged**